### DISCUSSION TITLE
Mario Kart GC Slide issue!

### DISCUSSION THREAD

**Discussion by jackmiller1986** [darthcloud/BlueRetro#624] [Q&A]
Hello, 

I habe an issue with the game Mario Kart Double Dash.

When I play alone the game , Fresh blue retro installed, the slide funktion by pressing R to slide in the game wont work on the first port.

When i play, with 2 player , the first wont work with slide in the game by pressing the r button, but on the second controlle it works normaly like an rigina gc controller, ich have nothing change in the settings so far. I have change to a new esp23 toooo, because i was thinking the old one works not correctly, what can it be that the slide function on player 2 in mario kart DD work, and on the first port not. Settings are all original, nothing change, fresh flash , new esp23

thx allot, i hope for help

**Comment by darthcloud** [comment]
You loaded the merged trigger preset?

https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#33---example-preset-load

**Reply by jackmiller1986** [reply]
yes and dont take effekt, on ps5 and ps4 controller, what me wondering , if you playing with two players , the secon player can slite, but the first player not, i have checket two esp32 , an my soldering all 4 ports works , its hw1

**Comment by jackmiller1986** [comment]
I have tested now since my poast here, i have testet luigis mansion worked the r2, zelda windwaker to, only mario kart player one , is no slide recommend , thats show me that merged trigger works, but not the r2 button for player one in Mario kart DD, Player 2 Mario Kart r2 slide button works. I have soldered another new esp32 in my cube. i have testet three diffrent Esp, all three the same happens.
